draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1089,19 +1089,20 @@ The nil UUID is special form of UUID that is specified to have all
 ~~~~
 {: title='Nil UUID Format'}
 
+A Nil UUID value can be useful to communicate the absence of any other UUID value in situations that otherwise require or use a 128-bit UUID.  A Nil UUID can express the concept "no such value here". Thus it is reserved for such use as needed for implementation-specific situations.
+
```

Comment:
I think it's helpful to provide some context as to why this is here.

Reply:
I've been wondering why they call this a "nil UUID" rather than a "null UUID" (which makes more sense to me), but I think that ship has already sailed.